src/model/guild/mod.rs

```diff
@@ -32,6 +34,8 @@ use parking_lot::RwLock;
 #[cfg(all(feature = "cache", feature = "model"))]
 use std::sync::Arc;
 #[cfg(feature = "model")]
+use crate::http;
```

Comment:
https://github.com/serenity-rs/serenity/pull/600#issue-286393882: `Unused import at src/model/guild/mod.rs:37`
Is this import required then if the compiler states that it's unused?
I don't see any usage of the http module in your code, so I'm wondering why you've added it.

Reply:
I didn't, did I?
Perhaps I checked out something incorrectly...
My bad